build-instructions.md

```diff
@@ -0,0 +1,14 @@
+Building of project
+------------------
+
+0. `cd demo/src/beyond-grammar/` -> `tsc beyond-grammar-plugin.ts`, for compiling beyond-grammar-plugin.ts to js
```

Comment:
Could you just call tsc demo/src/beyond-grammar/beyond-grammar-plugin.ts instead of changing dir?